packages/editor/src/client/blocks/types.ts

```diff
@@ -218,7 +220,17 @@ export interface PageTeaserLink {
   page: PageRefFragment
 }
 
-export type TeaserLink = ArticleTeaserLink | PeerArticleTeaserLink | PageTeaserLink
+export interface CustomTeaserLink extends BaseTeaser {
+  type: TeaserType.Custom
+  contentUrl?: string
+  properties?: TeaserMetadataProperty[]
```

Comment:
One more time about this.

Is it possible to have undefined properties, or we should always force array -> empty array?